<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServeurWidget</class>
 <widget class="QWidget" name="ServeurWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServeurWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>317</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelNumeroPort">
          <property name="text">
           <string>Num√©ro de port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxNumeroPort">
          <property name="maximum">
           <number>9999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonLancementServeur">
        <property name="text">
         <string>Lancement serveur</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelClients">
      <property name="text">
       <string>Clients</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEditClients"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuitter">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>ServeurWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
